src_debug_ui/systems.rs

```diff
@@ -2,7 +2,7 @@ use bevy::prelude::*;
 use rapier::pipeline::PhysicsPipeline;
 
 pub fn setup_ui(mut commands: Commands, asset_server: Res<AssetServer>) {
-    let font_handle = asset_server.load("../assets/FiraSans-Bold.ttf").unwrap();
+    let font_handle = asset_server.load(format!("{}/../assets/FiraSans-Bold.ttf", env!("CARGO_MANIFEST_DIR")).as_str());
```

Comment:
I'm not sure this will work if this is included in other projects. Is this only local for examples?

Reply:
Yes, it's only used by local examples.